.github/BOTMETA.yml

```
@@ -880,12 +880,26 @@ files:
   $modules/system/python_requirements_facts.py:
     maintainers: willthames
     ignore: ryansb
-  $modules/system/:
+  $modules/system/aix_devices.py:
```

I think if you just change
```suggestion
  $modules/system/aix:
```
and nothing else from the original state, it's already correct. It will cover all modules in `system/` that start with `aix`.

Also, simply removing people without giving them a chance to respond first isn't nice, so let's keep adejoux for now. Otherwise this PR will be blocked for a certain time when we wait for a reaction.

---

k, I didn't know about the "startswith" trick. One advantage of breaking it down is to have more specific labels, but that's not really a big deal, so I just removed that.

Trying to mention @adejoux - github does not autocomplete his ID, not sure what's the semantics in that, if any.

---

GitHub's ID autocomplete recently got worse and worse. I don't know what's happening, but it's definitely not related to adejoux :)